src/ALabel.cpp

```diff
@@ -22,8 +22,10 @@ ALabel::ALabel(const Json::Value& config, const std::string& name, const std::st
   if (config_["max-length"].isUInt()) {
     label_.set_max_width_chars(config_["max-length"].asUInt());
     label_.set_ellipsize(Pango::EllipsizeMode::ELLIPSIZE_END);
+    label_.set_property("single-line-mode", true);
```

Comment:
gtkmm provides a wrapper for this property, `label_.set_single_line_mode(true)`.

Reply:
Is there any advantage to doing that? The GtkLabel docs list the properties and not that wrapper, so using `set_property()` seems appropriate. The wrapper doesn't really do anything here.

Reply:
Mostly cosmetic. It's just gtkmm C++ style of doing things vs gtk pure C way.
By calling `set_x` instead of `set_property(x)` you are allowing compiler to check that the property exists (it's exposed by gtkmm bindings generator) and the argument type is correct.

Reply:
Makes sense. Changed it.